WikipediaUnitTests/CLLocationManagerMock.swift

```diff
@@ -0,0 +1,55 @@
+import Foundation
+
+/// Mock for `CLLocationManager`
+final class CLLocationManagerMock: CLLocationManager {
+
+    // MARK: - Override CLLocationManagerProtocol
+
+    var _heading: CLHeading?
+    override var heading: CLHeading? {
+        return _heading
+    }
+
+    var _location: CLLocation?
+    override var location: CLLocation? {
+        return _location
+    }
+
+    override func requestAlwaysAuthorization() { }
```

Comment:
Why are these overrides empty?

Reply:
Because in production CLLocationManager on new installation can require user interaction - permission prompt